.github/workflows/align_templates.yml

```diff
@@ -0,0 +1,28 @@
+name: Ensures template/ dir and config file are aligned.
```

Comment:
Reads kinda weird in the list of PR checks.

Maybe change to `Ensure ...`? Or something else if you prefer?

Reply:
https://github.com/SciTools/.github/pull/61/commits/8d30077e53f2d6a437f4fef4c12e216eeb781635